common/lib/xmodule/xmodule/modulestore/tests/factories.py

```diff
@@ -321,27 +321,39 @@ def check_sum_of_calls(object_, methods, maximum_calls, minimum_calls=1):
     Instruments the given methods on the given object to verify that the total sum of calls made to the
     methods falls between minumum_calls and maximum_calls.
     """
+
     mocks = {
         method: Mock(wraps=getattr(object_, method))
         for method in methods
     }
 
-    with patch.multiple(object_, **mocks):
+    if isinstance(object_, type):
```

Comment:
Are you certain that this statement detects that `object_` is a class? I tried this:
```
>>> class A: pass
...
>>> type(A)
<type 'classobj'>
>>> isinstance(A, type)
False
```